tests/db-tests/src/test/java/org/apache/activemq/artemis/tests/db/paging/PagingTest.java

```diff
@@ -424,8 +425,6 @@ public void testSimpleCursorIteratorLargeMessage() throws Exception {
       server = createServer(true, config, PAGE_SIZE, PAGE_MAX);
       server.start();
 
-      final int numberOfBytes = 200 * 1024;
```

Comment:
The method says it is for testing LargeMessage, yet it isnt using the numberOfBytes variable, and doesnt obliviously appear to be using large messages.

Rather than needing removal, this seems more like its indicating a test bug that needs fixed?

Reply:
I sent a new, separate commit to deal with this.